README.md

```diff
@@ -202,11 +244,12 @@ staramr db restore-default
 
 ## List of genes to exclude
 
-By default, the ResFinder/PointFinder genes listed in [genes_to_exclude.tsv][] will be excluded from the final results. To pass a custom list of genes the option `--exclude-genes-file` can be used, where the file specified will contains a list of the sequence ids (one per line) from the ResFinder/PointFinder databases. For example:
+By default, the ResFinder/PointFinder/PlasmidFinder genes listed in [genes_to_exclude.tsv][] will be excluded from the final results. To pass a custom list of genes the option `--exclude-genes-file` can be used, where the file specified will contains a list of the sequence ids (one per line) from the ResFinder/PointFinder/PlasmidFinder databases. For example:
 
 ```
 #gene_id
 aac(6')-Iaa_1_NC_003197
+ColpVC_1__JX133088
```

Comment:
Thanks for adding an example of how to exclude plasmid genes

Reply:
No problem! :smile: